SQL/create table.sql

```diff
@@ -0,0 +1,21 @@
+CREATE DATABASE phonebook;
+
+CREATE TABLE users (
+	client_name VARCHAR(20) NOT NULL,
+	email VARCHAR(30) UNIQUE NOT NULL,
+	passkey VARCHAR(20) NOT NULL,
+	USERID SERIAL PRIMARY KEY
```

Comment:
Rename this field to `id`. Always have an `id` field in your tables. Better not to get creative with it, because you will forget.